source/triggers/database-triggers.txt

```diff
@@ -166,17 +193,17 @@ Database Triggers have the following configuration options:
 
           Regardless of this setting:
           
-          - ``INSERT`` and ``REPLACE`` events always include the
+          - **Insert** and **Replace** events always include the
             ``fullDocument`` field.
           
-          - ``DELETE`` events never include the ``fullDocument`` field.
+          - **Delete** events never include the ``fullDocument`` field.
 
    * - :guilabel:`Document Preimage`
```

Comment:
My field's disabled with a note "You must be using an Atlas Cluster running MongoDB 4.4+ and a non-sharded cluster to use this feature." Do we mention the reqs for this feature anywhere?